### DISCUSSION TITLE
Is network drive / NAS a possible destination, and can we save per-device destination settings?

### DISCUSSION THREAD

**Discussion by alexlucasdev** [outdoorbits/little-backup-box#150] [Q&A] [answered]
I'd like to set up a "download station" for my kids and their digital cameras, so that they can have the following fancy flow:

- Take a bunch of pictures with their camera
- Plug camera into a raspberry pi
- Wait for "done" signal (Might get an LCD screen or USB stoplight or something for that)
- Unplug camera

And behind the scenes, the following logic is performed:
- Kid 1's camera (C1) gets plugged in
- Little-Backup-Box (LBB) recognizes C1.  Downloads photos from C1, uploads them to //nas/path/C1
- Kid 2's camera (C2) gets plugged in
- LBB recognizes C2.  Downloads photos from C2, uploads them to //nas/path/C2

Is this a thing that's currently possible with LBB?

**Comment by outdoorbits** [comment]
Did you already install lbb? If your kids cameras are not the same models, the backup paths will be different. Else you could configure your cameras to use different folders, this would also do the job.

Running a second job to perform a backup to network automatically is not implemented yet. But it seems to be a useful idea and not really a big deal. Would this workflow meet your needs:

- switch on lbb
- autostart backup camera to storage
- autostart storage to network
- shutdown

**Reply by alexlucasdev** [reply]
Yeah, that'd work great.  I don't have LBB installed yet, I wanted to ask before I went through all the setup and determined it was missing a feature :D Kids have the _exact_ same model camera, I'll have to check and see if there's an on-device way to set a different folder name.  Worst case scenario is that I could last-leg it myself and manually copy folders from a "landing zone" folder in the nas to individual kids' picture folders, but where's the fun in that :D

In case I need to do that:  Is it possible to have LBB automatically create a folder for each backup job?  Say date-time-stamped or something?  or is it just copying the folderit sees on the camera (say `DCIM` ) and then moving that around to intended destinations?

**Comment by outdoorbits** [comment]
The path is defined by camera and folders of the camera. Creating folders by timestamp would complicate cumulative backups a lot and provoke errors. Would be best to create unique names of folders or images by the camera.
Anyway I will add an option to define a secondary backup to web. Might take a few days.

**Comment by outdoorbits** [comment, marked as answer]
There is a new function: "Default secondary backup mode". You can define you primary default backup (camera to local storage) and a  secondary default backup to push your backup to a web-service.
The problem about the two identical cameras should be solved also. The backup-path now contains the serial-number of the device.
Please tell me about your experience.
